lib/sinon/util/core/iterable-to-string.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+var typeOf = require("../../typeOf");
+
+module.exports = function (obj) {
```

Comment:
I totally agree, I always name my functions whether they get assigned to a variable or are used only once, I just thought you prefer these functions to not have names due to what I've seen in other files.

Thanks for the feedback!